### DISCUSSION TITLE
Unattendend Install Issue

### DISCUSSION THREAD

**Discussion by sigmaphi007** [moudey/Shell#220] [General]
Hi, 

I used your application with NTLITE to post-setup unattended using the silent parameter, Your app opens the browser which leads to your website. 

This break makes unattended install kinda useless as I have to click the error to go away before the final stages boots up.

**Comment by sigmaphi007** [comment]
Here is the error I got 
![20230324_172655](https://user-images.githubusercontent.com/7560211/227585049-ea7b5523-2123-4f9b-b867-473ef1fa319b.jpg)

**Comment by moudey** [comment]
I will add a parameter that allows you to bypass opening online documentation during installation.

**Comment by sigmaphi007** [comment]
Hi, please any update on this? 
Thanks in advance!
